connector.go

```diff
@@ -107,7 +107,8 @@ func (c connector) Deploy(ctx context.Context, image string) (deployer.Plugin, e
 			_ = stdoutWriter.Close()
 			_ = stdinWriter.Close()
 		}()
-		_ = podExec.Stream(
+		_ = podExec.StreamWithContext(
+			ctx,
```

Comment:
Was this change required in order to be able to update the dependencies?

Reply:
yes, the previous function, `Stream()` was deprecated in the newer version, and the linter demanded the deprecated stuff be removed